woocommerce/wc-admin
# Save section/block preferences in user data

Fixes #2040.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/57089650-343a2280-6d05-11e9-8c90-af6151f687cb.png)

### Detailed test instructions:
- Modify section names and enable/disable some blocks.
- Reload the page.
- Verify changes are persisted.
